### PR TITLE
docs/work-with-flux

### PR DESCRIPTION
**Pull request by learner0810**
**What type of PR is this?**
/kind documentation

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:
Fixes https://github.com/karmada-io/karmada/issues/861

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

File: docs/working-with-flux.md [hunk 1]
```diff
@@ -0,0 +1,213 @@
+# Use flux support helm chart propagation
+
+[Flux](https://github.com/fluxcd/flux) is most useful when used as a deployment tool at the end of a Continuous Delivery pipeline. Flux will make sure that your new container images and config changes are propagated to the cluster.
+
+##Start up Karmada clusters
```
Comment (RainbowMango) [inline docs/working-with-flux.md:5]:
```suggestion
## Start up Karmada clusters
```

File: docs/working-with-flux.md [hunk 2]
```diff
@@ -0,0 +1,213 @@
...
+You just need to clone Karmada repo, and run the following script in Karmada directory. 
+
+```
+hack/local-up-karmada.sh
+```
+
+##Start up flux
```
Comment (RainbowMango) [inline docs/working-with-flux.md:12]:
```suggestion
## Start up flux 
```

**Review by RainbowMango — COMMENTED** [review]
Thanks @learner0810.
Generally look good to me, except the nits that `markdown` requires a `space` between `#` and title words.

**Comment by learner0810** [comment]
> Generally look good to me, except the nits that `markdown` requires a `space` between `#` and title words.

Thanks for the reminder, I've finished the revision, please take a look again

**Review by RainbowMango — APPROVED** [review]
/lgtm
/apprve

**Pull request merged**